packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -27,7 +27,7 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
     boxSide: 'start'
   };
 
-  private _checkBox = createRef<HTMLInputElement>();
+  private _checkBox = createRef<HTMLButtonElement>();
```

Comment:
Hmm. I think this is correct, but should we just make this more general? `HTMLElement` or something?

Reply:
Fixed